### PR TITLE
Refactored some components

### PR DESCRIPTION
**Pull request by sibela46**

### CODE REVIEW

**Review by tillywoodfield — APPROVED** [review]

File: src/components/Turn/Turn.jsx [hunk 1]
```diff
@@ -3,6 +3,8 @@ import socket from '../../socket';
 import axios from 'axios';
 import AllCards from '../Cards/AllCards'
 import Message from '../Message/Message'
+import PlayerCards from '../Cards/PlayerCards';
+import style from './Turn.module.css';
```
Comment (Ro4052) [inline src/components/Turn/Turn.jsx:7]: Some files use `styles` and some use `style`. You might want to keep this consistent.

File: src/components/Turn/Turn.jsx [hunk 2]
```diff
@@ -55,20 +56,21 @@ class Turn extends React.Component {
...
-                </ul>
-                <div>
-                    {`It's ${currentUsername}'s turn.`}
+            <div className={style.roundInfo}>
+                <h2>Round: {this.state.gameState.roundNum}</h2>
+                <div className={style.currentPlayersBox}>
+                    Players:
+                    <ul>
+                        {this.state.gameState.players.map((player, key) => {
+                            return <li key={key}>{player.username}</li>
```
Comment (Ro4052) [inline src/components/Turn/Turn.jsx:65]: This could be a single line, you don't need the return.

File: src/components/Turn/Turn.jsx [hunk 3]
```diff
@@ -55,20 +56,21 @@ class Turn extends React.Component {
...
+                        })}
+                    </ul>
                 </div>
+                <h2>{`It's ${currentUsername}'s turn.`}</h2>
```
Comment (Ro4052) [inline src/components/Turn/Turn.jsx:69]: Doesn't need to be a string.
```suggestion
                <h2>It's {currentUsername}'s turn.</h2>
```

File: src/components/Turn/Turn.module.css [hunk 4]
```diff
@@ -0,0 +1,15 @@
+.roundInfo {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    font-family: var(--main-font);
```
Comment (Ro4052) [inline src/components/Turn/Turn.module.css:6]: I like the idea of using variables : )

**Review by Ro4052 — COMMENTED** [review]
Looks good overall

**Pull request merged**